manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml

```diff
@@ -36,6 +36,12 @@ spec:
       {{- end }}
       affinity:
 {{ toYaml .Values.linux.affinity | indent 8 }}
+      securityContext:
+        {{- /* This is required for the driver to function correctly. */}}
+        privileged: true
```

Comment:
Why is `privileged: true` being moved from container to the pod security context?